### DISCUSSION TITLE
[✨] vite ssr mode: pre-load entry.ssr.tsx

### DISCUSSION THREAD

**Discussion by wmertens** [QwikDev/qwik-evolution#131] [Proposals For Qwik]
### Is your feature request related to a problem?

I would like to run server code inside the qwik-city server.
:point_right: Obviously I can put it in the server entry, but I also want to run in dev ssr mode.
:point_right: Obviously I can call the server entry point from `entry.ssr.tsx` which is called from both places, but then Vite will only execute that on first render.

### Describe the solution you'd like

Is it possible to load the entry code before vite calls it? That way, it also speeds up the first SSR render.

### Describe alternatives you've considered

Maybe an option to define a server-code entry point?

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
